app/src/androidTest/java/com/github/sdp/tarjetakuna/FilterCardsTest.kt

```diff
@@ -63,12 +63,13 @@ class FilterCardsTest {
     private val withIdReturnMainButton = withId(R.id.returnMainButton)
 
     @Before
-    public fun setUp() {
+    fun setUp() {
         val gson = Gson()
         val arrayCardJson = gson.toJson(arrayCard)
         val intent = Intent(ApplicationProvider.getApplicationContext(), FilterCardsActivity::class.java)
         intent.putExtra("cards", arrayCardJson)
         activityRule = ActivityScenario.launch(intent)
+        Thread.sleep(100) // Wait for the activity to be created (otherwise the test fails because there is no view)

```

Comment:
should not have Thread sleep there, but i know why you put it. 
put a todo to fix in a later time by checking that the scenario is actually launched correctly !

Reply:
True!